### PR TITLE
Add CI pipeline for building techpoc

### PR DESCRIPTION
**Pull request by siredmar**
This PR adds the CI/CD pipeline for building and deploying techpoc.

Tasks:
- [x] build torizon-builder
- [x] pipeline must check if 'project' on docker registry exists. If not create one.
- [x] build torizon yocto image
- [x] upload torizon yocto image to AWS S3 bucket
- [x] documentation on how to build techpoc via ci/cd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,4 +31,53 @@ To build and deploy the toradex image, simply call the appropriate dobi resource
...
+sudo chmod +x /usr/local/bin/
+```
+
+Then login to your concourse instance.
+
+```bash
+fly --target local login --concourse-url http://localhost:9090 -u <user> -p <password>
+```
+
+You can copy `ci/config.yaml` and `ci/credentials.template.yaml` and adapt it to your needs.
```
Comment (KlausPopp) [inline README.md:60]: File is credentials.example, not credentials.template

File: README.md [hunk 2]
```diff
@@ -28,4 +31,53 @@ To build and deploy the toradex image, simply call the appropriate dobi resource
...
+### Prerequisites
+
+-   Concourse CI, MinIO and Harbor e.g. via local-dev-environment (<https://github.com/ci4rail/local-dev-environment>)
+-   docker (<https://docs.docker.com/install/>)
+
+### Usage
+
+To build and deploy the toradex image, you must set the pipeline to the concourse CI server.
+
+First download `fly` from your concoruse server. The following example uses a local concourse server that is reachable at https://localhost:9000.
```
Comment (KlausPopp) [inline README.md]: typo "concoruse"

File: README.md (hunk 1 above)
Comment (siredmar) [reply README.md:60]: fixed

File: README.md (hunk 2 above)
Comment (siredmar) [reply README.md]: fixed

**Review by KlausPopp — APPROVED** [review]

**Pull request merged**